apps/src/p5lab/spritelab/SpritelabInput.jsx

```diff
@@ -128,26 +148,35 @@ class SpritelabInput extends React.Component {
           </div>
         );
         break;
-      case PromptType.MULTIPLE_CHOICE:
+      case PromptType.MULTIPLE_CHOICE: {
+        const choices = inputInfo.choices.filter(choice => !!choice);
         inputRow = (
           <div style={styles.inputRow}>
-            {inputInfo.choices.map(
-              choice =>
-                choice && (
-                  <button
-                    key={choice}
-                    style={styles.choiceButton}
-                    type="button"
-                    value={choice}
-                    onClick={this.onMultipleChoiceSubmit.bind(this)}
-                  >
-                    {choice}
-                  </button>
-                )
+            {choices.map((choice, index) =>
```

Comment:
naming nit: changing `choice` to `choiceKey` would be clearer here (unless i'm misunderstanding the shape of the `choices` array)

Reply:
`choice` is a string with the text (or name of the image) that will be shown, so I think `choice` makes more sense as a name than `choiceKey`. We do also use it (along with the index) as the key for React, but that's sort of secondary, I think.

Reply:
ah, okay. i assumed `choice` was a key because we are using it to key into `spriteMap`, but that makes sense